### DISCUSSION TITLE
Integration of yourls not working

### DISCUSSION THREAD

**Discussion by un99known99** [PrivateBin/PrivateBin#1327] [General]
Quick question, I am trying to integrate my local privatebin but cant get it run.
Config:
... tried various combinations ...

urlshortener = "https://myserver:34522/yourls-api.php?signature=345435435355345&action=shorturl="
or 
urlshortener = "https://myserver:34522/yourls-api.php?signature=345435435355345&?link="

not working, error

**Comment by rugk** [comment]
What kind of error? Please report this as a question category and fill out the issue template, the next time.

Also, did you already check-out the configuration docs? https://github.com/PrivateBin/PrivateBin/wiki/Configuration#urlshortener

If his does not help, please provide all options.

Edit: changed wiki link

**Comment by un99known99** [comment]
@rugk  just doing nothing, error page, I followed setup described in https://github.com/PrivateBin/PrivateBin/wiki/Configuration#urlshortener in various combination but either description there or yourls api desc is wrong

Could you show me your working urlshortener call plz?

**Reply by rugk** [reply]
See also https://github.com/orgs/PrivateBin/discussions/1182 for more discussion. Also, please post and/or (best: and) screenshot the error you are experiencing.
